### ISSUE TITLE
FR: new math fonts

### ISSUE THREAD

**Issue by mbertucci47** [fred-wang/MathFonts#18]
This is a FR to add some new-ish math fonts to the script (I see NewCM and LeteSans were added earlier today!).
- Erewhon/Fourier Math (https://www.ctan.org/pkg/erewhon-math)
- kpfonts (https://www.ctan.org/pkg/kpfonts-otf)
- Concrete (https://www.ctan.org/pkg/concmath-otf)
- XCharter (https://www.ctan.org/pkg/xcharter-math)
- Euler (maintained fork of Neo Euler; https://www.ctan.org/pkg/euler-math)
- Noto Sans (https://notofonts.github.io/math/)

**Comment by apoorvpotnis** [comment]
There's the Nagwa TK fonts by Khaled Hosny as well: https://fonts.nagwa.com/tk/

**Comment by mbertucci47** [comment]
@apoorvpotnis Yes and I also forgot Old Standard Math (https://www.ctan.org/pkg/oldstandard) which I see you also wrote in your TeX SX [answer](https://tex.stackexchange.com/a/711737/208544)

**Comment by apoorvpotnis** [comment]
This reminds me of IBM Plex as well: https://github.com/IBM/plex

It's nice to see so many opentype fonts cropping up in recent times.

**Comment by apoorvpotnis** [comment]
I've tried to add these fonts to but due to my illiteracy, I don't quite understand how to submit a pull request. This is the link the to folder I've tried to modify: https://drive.google.com/drive/folders/1Gt6awld5xIpgTTQmJI3lxUGHj79XgHnY?usp=sharing. Right now, I've only changed the css, as I don't quite understand how to modify the makefile.

**Comment by fred-wang** [comment]
Thanks everybody. I was not aware that so many new math fonts were released. Happy to accept PRs to add them.

**Comment by norbusan** [comment]
@mbertucci47 IBM Plex done, and Noto Sans PR sent.

**Comment by davidcarlisle** [comment]
Before noticing this issue we had added several fonts to a Neil Soiffer's fork, we didn't make a PR back as that code has deviated a bit, in particular modifying the CSS to make it easier to load multiple (or all) the fonts at the same time (possibly more useful for test documents than real documents) and to always use the web font for the math font not local versions (as problems arise with old or variant copies of stix being distributed and installed locally, so we wanted to ensure we were testing the versions in the repository.


The git repository has just been moved so it's no longer a fork in the github sense so it can have it's own issue tracker although it's still a git clone and we could potentially  resync at some point.

I think all the fonts mentioned above are there and you can see them side by side at

https://mathfonts.github.io/mathml-table.html
